web-server/django_web_server/website/server.py

```diff
@@ -41,22 +46,59 @@ def getClients(self):
             print("Server issues" + str(e))
             return []
 
-    def __saveTask(self, c_id, task_t, task_i):
-        client = Client.objects.get(client_id=c_id)
-        t = time.localtime()
-        asc_t = time.asctime(t)
-        client.senttask_set.create(start_time=asc_t, finish_time='ongoing', task_type=task_t, task_info=task_i)
+    def __saveTask(self, t_id, c_id, task_t, task_i, t_status):
+        if task_t != 'abort':
+            client = Client.objects.get(client_id=c_id)
+            t = time.localtime()
+            asc_t = time.asctime(t)
+            client.senttask_set.create(task_id=t_id, start_time=asc_t, status=t_status,
+                                       task_type=task_t, task_info=task_i)
+
+    def getTasks(self):
+        # TODO: Update when endpoint alltasks is implemented
```

Comment:
This endpoint is already implemented.

Reply:
Nice! We updated the code to only make one request to load all tasks

Reply:
Is the code updated? Then it's maybe possible to remove the TODO